src/components/StatisticBlock.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { FC, ReactNode } from 'react'
+import styled from 'styled-components'
+
+import LoadingSpinner from './LoadingSpinner'
+
+const Block = styled.div`
+  background-color: ${({ theme }) => theme.bgTertiary};
+  border: 1px solid ${({ theme }) => theme.borderSecondary};
+  box-shadow: ${({ theme }) => theme.shadowPrimary};
+  border-radius: 7px;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  padding: 2rem;
+`
+
+const Title = styled.div`
+  color: ${({ theme }) => theme.textSecondary};
+  font-size: 1.1rem;
+  margin-bottom: 1.5rem;
+`
+
+const Primary = styled.div`
+  color: ${({ theme }) => theme.textPrimary};
+  font-size: 2rem;
+  margin-bottom: 0.3rem;
+`
+const Secondary = styled.div`
+  color: ${({ theme }) => theme.textSecondary};
+`
+
+interface Props {
+  title: string
+  primary: ReactNode
+  secondary: ReactNode
+  isLoading: boolean
+}
+
+const StatisticBlock: FC<Props> = ({ title, primary, secondary, isLoading }) => {
```

Comment:
As we don't use `children`, we don't need to use `FC`. We can just go like: 

```ts
const StatisticBlock = ({ title, primary, secondary, isLoading }: Props) => 
```

And as this is a "dump component" (presentational only, no logic), we don't need the brackets and the return statement :)

Reply:
Habit to just make everything FC<>, even if a dumb component, will remove :+1:

Reply:
Gah, I still need to do this.

I would still use `FC` but you're right about removing the `return` - it's 2 characters for a more accurate type. If you feel strongly about omitting it I'll do it though, I just don't see any good reason other than "it adds more semantics to the code", which is a good reason I guess but not one I agree with :slightly_smiling_face:

Reply:
I haven't seen any benefit of having a `FC` over a normal function so far. So we decided to drop it when not needed and save one line of imports.

Reply:
Fair enough! I'll omit it.

----

The comparison I see is you wouldn't just leave a function untyped normally, because it's bad type safety: `const f = (s: string): string => 'takes and returns a string';`
So I don't (personally) compromise when having a function which returns an element either: `const F = (p: Props):ReactElement => <></>;` - `FC` is just shorthand for this. You never see this as an issue though because TypeScript is inferring that `<></>` is an `Element` so it deduces that `F` returns an `Element`, but still, it's not good because what if you made a mistake in the function which returns a `string` or a type other than `Element`?